Dockerfile.drivers

```diff
@@ -8,23 +8,23 @@
 # license agreement from NVIDIA CORPORATION is strictly prohibited.
 #
 
-ARG CUDA_VERSION=9.2
+ARG FROM_IMAGE=gpuci/clara-genomics-base
+ARG CUDA_VER=9.2
 ARG LINUX_VERSION=ubuntu16.04
 ARG CC_VERSION=5
-ARG CXX_VERSION=5
 ARG PYTHON_VERSION=3.6
 
-FROM gpuci/rapidsai-base-driver:cuda${CUDA_VERSION}-${LINUX_VERSION}-gcc${CC_VERSION}-py${PYTHON_VERSION}
+FROM ${FROM_IMAGE}:cuda${CUDA_VER}-${LINUX_VERSION}-gcc${CC_VERSION}-py${PYTHON_VERSION}
 
-# Install htslib dependencies
-RUN apt-get update
-RUN apt-get install -y tabix \
-        zlib1g-dev \
-        libbz2-dev \
-        liblzma-dev \
-        libcurl4-gnutls-dev \
-        wget \
-        libssl-dev      # VariantWorks `cyvcf2` dependency
+# Installs cuda-drivers and cuda libraries for conda builds on CPU-only machines
 
-# Install htslib
-RUN wget https://github.com/samtools/htslib/releases/download/1.9/htslib-1.9.tar.bz2 && tar xvf htslib-1.9.tar.bz2 && cd htslib-1.9 && ./configure && make -j16 install
+# Required arguments
+ARG DRIVER_VER="440"
+
+# Update and add pkgs
+RUN apt-get update -q \
```

Comment:
just curious why this one needs the `headless` components whereas the other ones don't?

Reply:
There are actually two ways to build the driver images, we can either be based on the image we just created in `gpuci/clara-genomics-base` (`./Dockerfile`). or we can be based on `gpuci/miniconda-cuda-driver` (which already contains this logic) and copy the content of  `./Dockerfile`, I went with the former.

Reply:
Yep I think that's totally fine. I was just curious why this particular image needed this `headless` thing. Maybe that's a question more for @raydouglass

Reply:
I see these commands were updated in https://github.com/rapidsai/gpuci-build-environment/commit/4d4337f44f2f90c420ae1e323d5fd97674ca75d3

Reply:
but that change shows those dependencies were removed?

Reply:
@tijyojwad I changed the driver image to be built based on `gpuci/miniconda-cuda-driver` and not `gpuci/clara-genomics-base` since different CUDA versions require a different driver installation as in https://gpuci.gpuopenanalytics.com/job/gpuci/job/docker/job/miniconda-cuda-driver/
so we don't need to reconfigure https://gpuci.gpuopenanalytics.com/job/clara-genomics/job/docker/job/clara-genomics-base-driver/ for now

Reply:
Awesome, thanks!